docs/index.md

```diff
@@ -3,87 +3,87 @@ hide:
-- [**"User Manual"**](manuals/about.md) describes the main concepts for Taipy GUI, Taipy Core and Taipy REST. It also  provides information on the deployment of your Taipy app.
-- [**"Reference Manual"**](manuals/reference) documents all the Taipy APIs.
+For data scientists and developers alike, Taipy acts as a catalyst for successful Python endeavors. Whether your aim is a straightforward pilot or a comprehensive application within IDEs or Notebooks, Taipy equips you with all indispensable functionalities. Its architecture is meticulously crafted to truncate both development and deployment timelines significantly.
 
-Taipy requires *Python 3.8* or newer.
+Taipy's web application builder is a robust tool that empowers individuals and enterprises to develop web applications without extensive coding expertise. A primary advantage of Taipy lies in its ability to expedite development across both front-end and back-end domains, encompassing everything from initial prototypes to fully scalable, production-ready applications.
 
-## Quick Access
+**Taipy Front-End Functionalities:**
 
-These topics are the most visited ones, we thought you’d like to have a glance at them!
+- Crafting a user interface is accomplished with a foundational understanding of Python programming.
+- Taipy is purposefully engineered to prioritize user-friendliness, resulting in a straightforward and intuitive process of user interface creation.
+- No prior knowledge of web design is necessary, and it eradicates the need for any prerequisites related to CSS and HTML.
+- Leveraging augmented markdown syntax, Taipy aids users in seamlessly generating their desired web pages.
 
-<div class="tp-row tp-row--gutter-sm">
-  <div class="tp-col-12 tp-col-md-6 d-flex">
-    <a class="tp-content-card" href="getting_started/">
-      <header class="tp-content-card-header">
-        <img class="tp-content-card-icon icon-light" src="images/icons/flag-w.svg">
-        <img class="tp-content-card-icon icon-dark" src="images/icons/flag.svg">
-        <h3>Getting Started</h3>
-      </header>
-      <p>
-        This tour shows you how to create an entire application using the two main components of Taipy.
-      </p>
-      <span class="tp-content-card-readmore">Read more</span>
-    </a>
+**Taipy Back-End Functionalities:**
+
+- Taipy provides extensive support for establishing resilient pipelines capable of managing diverse scenarios.
+- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through Taipy's functionality.
+- Enhancing the overall performance of Taipy applications, the data caching feature is integrated.
+- Facilitating a registry of pipeline executions.
+- Enabling pipeline versioning.
+- Taipy equips users with the ability to monitor and assess the performance of their applications using the KPI tracking tool.
+- Furthermore, Taipy offers a built-in visualization feature for pipelines and their associated data.
+
+## Overview of the Taipy Flow
```

Comment:
I see "Overview of the Taipy Flow", which is used in general for topics where you display Schemas / Diagrams. Not sure at the moment what else we could add there?

Reply:
I'm not proposing to add anything.
But I have a problem with the generic term "flow".
As if there was a natural flow to use Taipy. 
This might be the case for Core or REST (where you design the configs then create the app that deals with entities...), but certainly not with GUI, or any other piece of the puzzle...
That was my only point.